### DISCUSSION TITLE
Happy and Sad at the same time...

### DISCUSSION THREAD

**Discussion by Balth021** [mtalcott/google-photos-deduper#59] [General]
I woke up at 4:30 am this morning planning on studying Python to finish my last class in my degree.  Yesterday, I imported a ton of photos from my Google Drive to my desktop only to realize I had so many duplicates, which led me this morning to do a quick Google search...which led me to the Reddit post...which led me here. 

Almost 4 hours later, after lots of ChatGPT questions, frustration, and coffee, I finally deleted 500+ duplicates. 

Did I do what I planned for this morning?  Nope.  
Did I scratch an internal itch?  Yep. 

Gather media items (26,973 in 19:56)
Process duplicates (448 groups in 04:08)

Appreciate the creation and write-up.  This is the first time I've touched many of these things, and it wasn't too painful. 

Also, I'm unfamiliar with GitHub and "push to main." Does that try to push to my code here???? lol. 
So much to learn, one step at a time.

**Comment by aleon1220** [comment]
You need to learn a little bit of python3, Docker containers, docker compose, Cloud computing, GCP and google API concepts.
My setup was done with a Linux ubuntu machine in under 15 minutes though i have work experience in all topics mentioned above. 

Your last line has to do with the GitHub flow of pushing changes to the **main** git branch of code. Unless you are suggesting changes to the code, you dont need to know that.
